pymeasure/instruments/hp/hp8753e.py

```diff
@@ -0,0 +1,390 @@
+    )
+
+    power = Instrument.control(
+        "POWE?",
+        "POWE%.3e",
+        """Control the output RF power of the instrument's active port. (float truncated from -70.0 to 10.0 in dBm).""",
+        cast=float,
+        validator=truncated_range,
+        values=[-70, 10],
+    )
+
+    output_enabled = Instrument.control(
+        "SOUP?",
+        "SOUP%s",
+        """Control RF Output State (boolean)""",
+        cast=bool,
+        map_values=True,
+        validator=strict_discrete_set,
+        values={True: 1, False: 0, "ON": 1, "OFF": 0},
+    )
+
+    trigger_hold = Instrument.control(
+        "HOLD?",
+        "%s",
+        """Control Sweep Scan Trigger State (boolean)""",
+        cast=str,
+        # map_values=True,
+        # validator=strict_discrete_set,
+        set_process=lambda x: "HOLD" if x else "CONT",
+        get_process=lambda x: x == "1",
+    )
+
+    trigger_continuous = Instrument.control(
+        "CONT?",
+        "%s",
+        """Control Sweep Scan Trigger State (boolean)""",
+        cast=str,
+        # map_values=True,
+        # validator=strict_discrete_set,
+        # values={True: "CONT", False: "HOLD"}, # , "1": True, "0": False, 1: True, 0: False},
+        set_process=lambda x: "CONT" if x else "HOLD",
+        get_process=lambda x: x == "1",
+    )
+
+    scan_points = Instrument.control(
+        "POIN?",
+        "POIN%d",
+        f"""Control the number of points used for a scan sweep. From the set [{[f'"{value}"' for value in SCAN_POINT_VALUES]}]""",
+        cast=lambda x: int(float(x)),
+        validator=strict_discrete_set,
+        values=SCAN_POINT_VALUES,
+    )
+
+    id = Instrument.measurement(
+        "*IDN?",
+        """Get the identification of the instrument""",
+        cast=str,
+    )
+
+    sn = Instrument.measurement(
+        "OUTPSERN",
+        """Get the serial number for the instrument""",
+    )
+
+    options = Instrument.measurement(
+        "OUTPOPTS",
+        """Get the installed options for the instrument""",
+    )
+
+    @property
+    def manu(self):
+        """Get the manufacturer of the instrument."""
+        if self._manu == "":
+            self._manu, self._model, _, self._fw = self.id
+        return self._manu
+
+    @property
+    def model(self):
+        """Get the model of the instrument."""
+        if self._model == "":
+            self._manu, self._model, _, self._fw = self.id
+        return self._model
+
+    @property
+    def fw(self):
+        """Get the firmware of the instrument."""
+        if self._fw == "":
+            self._manu, self._model, _, self._fw = self.id
+        return self._fw
+
+    def set_fixed_frequency(self, frequency):
+        """Set the sweep to be a fixed frequency in Hz. (float truncated from 30_000.0 to 6_000_000_000.0)"""
+        self.start_frequency = frequency
+        self.stop_frequency = frequency
+        self.scan_points = 3
+
+    @property
+    def parameter(self):
```

Comment:
maybe "scattering_parameter" or even "active_scattering_parameter".

Could you do it with a property creator (`Instrument.control`)?

Reply:
I can switch to "measurement_parameter" to allow both scattering parameters and absolute measurements like A, B, C, D, R1, R2, R3, R4 as used in other VNAs like the E5061/E5071.

Reply:
Measurement parameter sounds fine too.

Reply:
Marking this one as resolved since there is another request higher in the comments requesting the implimentation gets changed to `Instrument.control`